### ISSUE TITLE
experiment details

### ISSUE THREAD

**Issue by dreamcubeblock** [klightz/PSF#7]
Hi, how many epochs do I have to train to achieve acceptable results? Could you upload a train log to see the loss changes?

**Comment by klightz** [comment]
Hi! I believe for first stage ~200 epochs is enough (things are the same
for the original PVD). For reflow and distillation stage, if you samples m
times size of the training set, you can fine tune it for something like
200/m epochs. For example, in our original setup, we sample the reflow set
10 times larger than training set so we only fine tune it for 100 epochs.

Sorry the training log was saved in the internal server so it is not easy
to export it out, if you have trouble training it with 200 epochs, please
let me know.

**Comment by dreamcubeblock** [comment]
It works, thanks!
